src/multivariate/mvnormal.jl

```diff
@@ -256,7 +256,7 @@ sqmahal(d::MvNormal, x::AbstractVector) = invquad(d.Σ, broadcast(-, x, d.μ))
 sqmahal!(r::AbstractVector, d::MvNormal, x::AbstractMatrix) =
     invquad!(r, d.Σ, broadcast(-, x, d.μ))
 
-gradlogpdf(d::MvNormal, x::Vector) = -(d.Σ \ broadcast(-, x, d.μ))
+gradlogpdf(d::MvNormal, x::AbstractVector) = -(d.Σ \ broadcast(-, x, d.μ))
```

Comment:
Should `x` be `x::AbstractVector{<:Real}`?

Also, style, `broadcast(-, x, d.μ)` -> `(x .- d.μ)`?